### ISSUE TITLE
Where is the azure-mgmt-compute documentation?

### ISSUE THREAD

**Issue by fjaguero** [Azure/azure-sdk-for-node#1389]
Hi,
I have been looking for more info about azure-mgmt-compute but I only have the example from the README. Is there any place to find a detailed documentation?

Thanks.

**Comment by jeffwilcox** [comment]
Have you looked inside the `lib/services` directories? Here's an example file that has at least a little more information: https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/computeManagement

For better or worse a lot of the Node.js SDK for Azure is delegated to code reading and folder-specific files, given the immense size and scope of the project.

Does this answer your question, @fjaguero ?

**Comment by fjaguero** [comment]
Thanks @jeffwilcox. There is some information there. The bad thing is that is not entirely documented but at least is a start point.

**Comment by amarzavery** [comment]
@fjaguero - We published a new version of the compute management library for the ARM (V2) version of Azure API. The documentation from the generated code will be out once the Autorest branch is merged into master. The new version of the code is modularized and is relatively easy to understand.  For now you can take a look at the generated code over here: https://github.com/Azure/azure-sdk-for-node/tree/autorest/lib/services/computeManagement2/lib
